wiki/Article_Style_Guide/en.md

```diff
@@ -101,9 +114,18 @@ or
 You can choose either the underline style or hashtag style of level 1 and level 2 headings.
 Note that heading levels 3 and onwards use the hashtag style.
 
+Optionally, but preferably, add an extra line before and after the headings.
+Doing so will help make the headings stand out a bit more when editing.
+
 ### Article Titles
 
+All articles must only have **one** level 1 heading that of which has to be the article's title.
```

Comment:
considering to change to `All articles must have **one** level 1 heading, being the article's title.`